pycontrails/core/models.py

```diff
@@ -696,12 +723,13 @@ def interpolate_met(
     vector: GeoVectorDataset,
     met_key: str,
     vector_key: str | None = None,
+    q_method: str = "linear-q",
```

Comment:
You could remove this input if you put this value in `interp_kwargs`.  You could then `pop` the key out so that the interface does not break downstream.

Reply:
fe10a1df11aed5cedc1afe767701ef1a869b7059